programming_challenges/hacker_rank/filter-array.clj

```diff
@@ -1,10 +1,9 @@
 ;; https://www.hackerrank.com/challenges/fp-filter-array/problem
 
 (defn filter-array
-  [n list]
-  (if (< (first list) n)
-    (println (first list)))
-  (if (not (empty? (rest list)))
```

Comment:
Nice improvement removing the `println`, it's encouraged to return the desired value instead of using `println`.